types/encoding-japanese/index.d.ts

```diff
@@ -69,7 +69,7 @@ export interface ConvertUnknownOptions {
 
 export function detect(data: RawType | string, encodings?: Encoding | Encoding[]): EncodingDetection;
 export function convert(data: RawType, to: Encoding, from?: Encoding): number[];
-export function convert(data: string, to: Encoding, from?: Encoding): string;
+export function convert(data: string | number[], to: Encoding, from?: Encoding): string;
```

Comment:
Instead of adding `number[]` here, would you add it to `IntArrayType` definition?

```typescript
type IntArrayType =
    | ReadonlyArray<number>
    | Uint8Array
    | Uint16Array
    | Uint32Array
    | Int8Array
    | Int16Array
    | Int32Array
    | number[];
```

Reply:
sorry my bad i will close PR